types/src/front/user.ts

```diff
@@ -31,6 +31,8 @@ export type LightWorkspaceType = {
   name: string;
   role: RoleType;
   segmentation: WorkspaceSegmentationType;
+  whiteListedProviders?: string[];
```

Comment:
could we make it 
```
whiteListedProviders: string[] | null;
defaultEmbeddingProvider: string | null;
```

in case we forgot to set it in other places we'll get a typescript error

Reply:
Good to know 🙏🏼